source/index.html.erb

```diff
@@ -68,7 +68,7 @@ description: GOV.UK Pay is for central government, local authorities, police and
         <div class="govuk-grid-column-two-thirds">
           <h2 class="content-section__title govuk-heading-m">What’s new</h2>
           <p class="govuk-body">We’re updating GOV.UK&nbsp;Pay all the time.</p>
-          <p class="govuk-body">Read the  <%= link_to 'full roadmap', 'roadmap', class: 'govuk-link' %>  to see what’s changed and what’s coming next.</p>
+          <p class="govuk-body">Read thenbsp;<%= link_to 'full roadmap', 'roadmap', class: 'govuk-link' %>nbsp;to see what’s changed and what’s coming next.</p>
```

Comment:
Please forgive me if I'm wrong, but don't you need ampersands before the nbps; ? Like

`Read the&nbsp;`

?

Reply:
yes 🤦